### DISCUSSION TITLE
How to save scholarly.pprint output to a variable?

### DISCUSSION THREAD

**Discussion by BetaUliansyah** [scholarly-python-package/scholarly#491] [Q&A]
I want to process more publication titles (retrieve and save to file). But I meet difficulty to save scholarly.pprint output to a variable. How do I achive this? Thank you in advance.

**Comment by arunkannawadi** [comment]
You should never try to save anything that comes from `pprint` to a variable. It's only meant to pretty print things. The `author` or `pub` variable should already have all the info you need. If you need to save to a file and retrive them, you should be able to save them as a `json` file using the native `json` package.
